static/css/styles.css

```diff
@@ -352,28 +357,43 @@ form[name=login_user_form] ul {
     z-index: 1;
 }
 
-form[name=login_user_form] ul li {
+form[name=login_user_form] ul li,
+form[name=change_password_form] ul li,
+form[name=register_user_form] ul li,
+form[name=forgot_password_form] ul li {
     text-align: center;
     font-size: 1.2rem;
     color: red;
 }
 
-form[name=login_user_form] ul:after, form[name=login_user_form] ul:before {
+form[name=login_user_form] ul:after, form[name=login_user_form] ul:before,
```

Comment:
new line after comma for having each selector on its line like [you did on the `:hover`](https://github.com/cafebabel/cafebabel.com/pull/78/files/edc0ab8265da5807c9bfc11bcf32cabb66cff046..df19f95d22a43b5ff7e1c16fe6377f330346591c#diff-7962f8c018310ce9bd37011f4b6ce5e3R390); easier to read in the longer term

Reply:
ok